components/pipeline-service/production/stone-prd-m01/resources/tekton-chains-signing-secret-path.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/dataFrom/0/extract/key
+  value: production/pipeline-service/stone-stage-m01/chains-signing-secret
```

Comment:
wrong key path? there is stage in the name of the secret

Reply:
Fixed